packages/nbconvert-css/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@jupyterlab/nbconvert-css",
+  "version": "0.0.1",
```

Comment:
As a general rule, we start packages at 0.1.0 so that we can issue patch releases without breaking things.

Reply:
Fixed!